linode_api4/groups/lke_tier.py

```diff
@@ -0,0 +1,40 @@
+from linode_api4.groups import Group
+from linode_api4.objects import TieredKubeVersion
+
+
+class LKETierGroup(Group):
```

Comment:
This is a new pattern since the LKE tier path is the first of its kind. Let me know if you have any feedback!